cli/spec/kontena/cli/master/use_command_spec.rb

```diff
@@ -26,13 +26,13 @@
       subject.run(['some_master'])
     end
 
-    it 'should fetch grid list from master' do
+    it 'should fetch grid list from master if invalid master name given' do
```

Comment:
I reverted this behaviour to the original. (But also added a new test for the failure case)